packages/api/__tests__/api/private/block.graphql

```diff
@@ -74,6 +74,15 @@ fragment FullBlock on Block {
     text
     linkText
     linkURL
+    linkTarget
```

Comment:
You don't need to update the tests if you don't intend to test the changes.

Reply:
yep

Reply:
would you like to add tests? does it harm?

Reply:
Not yet ready for testing blocks.

Reply:
you introduced testing, which is still in development, i suggest you add this to understand the block extension better

Reply:
Just remove it.